glide.yaml

```diff
@@ -10,7 +10,7 @@ import:
 - package: github.com/karlseguin/ccache
 - package: github.com/spf13/pflag
 - package: k8s.io/kubernetes
-  version: v1.2.4
+  version: v1.3.0
```

Comment:
I usually prefer to spread additional changes to different PRs to avoid batching changes but no worries, killing two birds with one stone 👍